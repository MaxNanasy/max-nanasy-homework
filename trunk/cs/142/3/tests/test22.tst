type newType = short;
var newType : newType;
const newType2 = 10;
   void newType (int a, short b, newType2 c) {
   PRINT SYMBOL TABLE
   }
