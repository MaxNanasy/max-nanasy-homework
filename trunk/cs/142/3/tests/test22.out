Error: SEMANTIC:  UNDEFINED SYMBOL
***** Symbol Table Contents *****
N: int C: TYPE T: null
N: void C: TYPE T: null
N: short C: TYPE T: null
N: newType C: TYPE T: short
N: newType C: VAR T: newType
N: newType2 C: CONST T: short V: 10
N: newType C: PROCEDURE T: void
-->N: a C: VAR T: int
-->N: b C: VAR T: short
